packages/common/types/resolver.types.ts

```diff
@@ -0,0 +1,11 @@
+export interface ResolvedField {
+  [x: string]: ResolvedValidator[];
+}
+
+export type Kind = "max" | "min" | "required" | "email";
```

Comment:
I think this is more appropriate in the file validator.types.ts

Then maybe we can name it `ValidationType`